src/components/oauth/OAuthCodeHandler.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @author sarahr
+ */
+
+import React, { useCallback, useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { useQuery } from "react-query";
+import { build } from "@cyverse-de/ui-lib";
+
+import {
+    APP_CATEGORIES_QUERY_KEY,
+    getPrivateCategories,
+} from "serviceFacades/apps";
+import {
+    OAUTH_CALLBACK_QUERY_KEY,
+    doOAuthCallback,
+} from "serviceFacades/oauth";
+
+import OAuthErrorHandler from "./OAuthErrorHandler";
+import OAuthLinkingMessage from "./OAuthLinkingMessage";
+import ids from "./ids";
+import { getListingPath } from "components/apps/utils";
```

Comment:
Importing `getListingPath` directly from `components/apps/utils.js` seemed reasonable to me, but I'm curious about what others think.